### PR TITLE
Updates welcome page responsiveness

### PR DESCRIPTION
**Pull request by v-bbrady**
This PR fixes the responsive layout on the welcome page per direction of the ADUX team. It adjusts some margins, header line heights, and page breaks.

### CODE REVIEW

**Comment by alanrenmsft** [comment]
@v-bbrady screenshots?

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default () => `
 	<div class="welcomePage">
 		<div class="ads-homepage splash">
 			<div class="gradient">
+				<div class="content-container">
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:16]: nit: please indent the inner content properly to clearly be able to tell what is contained within this div

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 2]
```diff
@@ -22,12 +23,13 @@ export default () => `
...
-								<div class="icon sm"></div>
-								<div class="title">
-									<div class="caption-container">
-										<span class="icon xs"></span><h1 class="caption"></h1>
-									</div>
-									<div id="welcome-page-button-container" class="flex btn-container">
+								<div class="caption-container" style="display: flex;">
+									<div class="icon"></div>
+									<div class="flex flex-d-column">
+										<h1 class="caption flex flex-w-wrap">Azure Data Studio Dev</h1>
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:29]: Why is this hardcoded? And have "Dev" in it?

File: src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts [hunk 3]
```diff
@@ -326,13 +327,17 @@ class WelcomePage extends Disposable {
 			workspaces = workspaces.filter(recent => !this.contextService.isCurrentWorkspace(isRecentWorkspace(recent) ? recent.workspace : recent.folderUri));
 			if (!workspaces.length) {
 				const recent = container.querySelector('.welcomePage') as HTMLElement;
+				const moreRecent = document.querySelector('.moreRecent') as HTMLElement;
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts:330]: This ideally shouldn't be at the document level - can't you scope it down further?

**Comment by alanrenmsft** [comment]
@v-bbrady any update?

**Comment by kburtram** [comment]
@v-bbrady there are a few comments that need to be resolved to merge this.  e.g. hardcoded string "Dev", etc.  Do you have plans to make these updates, or is this PR abandoned?

**Comment by v-bbrady** [comment]
> 
> 
> @v-bbrady there are a few comments that need to be resolved to merge this. e.g. hardcoded string "Dev", etc. Do you have plans to make these updates, or is this PR abandoned?

Hey @kburtram I'll start on these updates today. For some reason I never got any emails letting me know of the initial comments until you tagged me in your comment. Thanks you.

**Comment by v-bbrady** [comment]
> @v-bbrady any update?

@alanrenmsft The updates have been made

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 4]
```diff
@@ -13,68 +13,71 @@ export default () => `
 	<div class="welcomePage">
 		<div class="ads-homepage splash">
 			<div class="gradient">
-				<div class="ads-homepage-section tool-tip">
-					<div class="tool-tip-container" id="tool-tip-container-wide">
-						<a role="img" tabindex=0 class="ads-welcome-page-link" aria-label="${previewImgDescription}" title="${previewImgDescription}" id="preview-link-wide" class="preview-link" tabindex="0" name="preview"><p>Preview</p><i class="icon-info themed-icon"></i></a>
+				<div class="content-container">
+					<div class="ads-homepage-section tool-tip">
+						<div class="tool-tip-container" id="tool-tip-container-wide">
+							<a role="img" tabindex=0 class="ads-welcome-page-link" aria-label="${previewImgDescription}" title="${previewImgDescription}" id="preview-link-wide" class="preview-link" tabindex="0" name="preview"><p>Preview</p><i class="icon-info themed-icon"></i></a>
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:19]: Preview should be localized

**Review by Charles-Gagnon — APPROVED** [review]

**Review by kburtram — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**